lib/vagrant-service-manager/command.rb

```diff
@@ -192,8 +192,16 @@ def execute_docker_info
 
           api_version = ""
           docker_apiversion = "docker version --format '{{.Server.APIVersion}}'"
-          machine.communicate.execute(docker_apiversion) do |type, data|
-            api_version << data.chomp if type == :stdout
+          if machine.communicate.test(docker_apiversion)
+            machine.communicate.execute(docker_apiversion) do |type, data|
+              api_version << data.chomp if type == :stdout
+            end
+          else
+            # fix for issue #152
+            docker_apiversion = "docker version --format '{{.Server.ApiVersion}}'"
+            machine.communicate.execute(docker_apiversion) do |type, data|
+              api_version << data.chomp if type ==:stdout
+            end
           end
```

Comment:
Probably you would like to get the `docker_api_version` based on `test` condition and later execute. Then, `execute` block will come once.